charts/oes/values.yaml

```diff
@@ -977,8 +977,9 @@ spinnaker:
     repo:
       type: git # git, S3, vault, only git support at this point
       organization: OpsMx
-      halRepository: gitops-haldir # Please provide the repo URL of the GitOps Halyard Directory.
-      dynamicAccRepository: standard-dynamic-account # Please provide the repo URL of the GitOps Dynamic Accounts Directory.
+      Repository: standard-gitops-repo # Please provide the repo URL of the GitOps Halyard Directory.
```

Comment:
this should be 'repository' in smaller case and the let's give the value in the form of a url, not as invalid keyword. Let's give our sample repos that we created by default in here.

https://github.com/OpsMx/gitops-hal-sample.git
https://github.com/OpsMx/gitops-accounts-sample.git

Reply:
We need to use this url in multiple places and in some cases we need it to enter along with user name and token, so we can not make in the form of url. The same is already present in our existing version, its just updated now with one parameter.

Reply:
Having it as url gives user an indication of how url should be. Otherwise, let's add detailed comments please

Reply:
Reason why I'm specifically stressing on specifying sample url is, if user doesn't keep ".git" suffix at the end, it won't clone.

Reply:
.git part is defined in the templates.

Reply:
Let's go with eith "null" or a valid input that's indicative

Reply:
@abhinaybyrisetty  have added new commit with the updated comments, please validate